crates/routes/src/images.rs

```diff
@@ -162,3 +150,9 @@ async fn delete(
 
   Ok(HttpResponse::build(res.status()).body(BodyStream::new(res)))
 }
+
+fn pictrs_url() -> Result<String, LemmyError> {
+  Settings::get()
+    .pictrs_url
+    .ok_or_else(|| anyhow!("images_disabled").into())
+}
```

Comment:
This should be in the settings somewhere, and maybe just be an Option so you can use that smart default.